autotest/interface/restful/test_restful_interface_func_turbomind.py

```diff
@@ -88,81 +88,14 @@ def test_chat_completions_max_tokens_stream(self):
             'finish_reason') == 'length'
         assert len(outputList) == 8
 
-    def test_chat_completions_repetition_penalty_stream(self):
```

Comment:
what's the reason for removing this case?

Reply:
> what's the reason for removing this case?

move to common testcase. the common case will run both in turbomind and pytorch backend.

Reply:
Currently, the behaviour is not the same of turbomind and pytorch backend. In the future, I think only maintain common testcases for both two backends. 
I found this case can be execute successfully in turbomind and pytorch backend, so move it to common testcase